### PR TITLE
Workspaces upgrade

### PR DESCRIPTION
**Pull request by kenk2**
### Summary

**What was changed:**
The build process now utilizes NPM workspaces.

**Why it was changed:**
To keep up with the node / npm ecosystem.

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [x] Other (please describe below)
- [ ] No tests are needed

Running and building the application without lerna and utilizing NPM workspaces.
### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-XXXX 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,13 +4,16 @@
   "version": "5.25.0",
   "description": "Utilities to help when developing terra modules.",
   "engines": {
-    "node": "10 || 12|| 14",
+    "node": "10 || 12 || 14",
     "npm": "6 || 9"
```
Comment (sdadn) [inline package.json]: Since workspaces is a npm 9 feature, we should remove npm 6.

File: package.json [hunk 2]
```diff
@@ -73,13 +76,12 @@
...
   },
   "scripts": {
-    "bootstrap:hoist": "lerna bootstrap --hoist --force-local",
     "clean": "npm run clean:root && npm run clean:lib",
     "clean:install": "rm package-lock.json; npm run clean && time npm install",
     "clean:lib": "rm -rf packages/**/lib",
     "clean:root": "$(lerna clean --yes || true) && echo 'removing ./node_modules' && rm -rf node_modules && echo 'removed ./node_modules'",
     "compile:prod": "rm -rf ./build; NODE_ENV=production webpack --mode=production --config webpack.dev-site.config.js",
-    "compile": "npm run clean:lib && lerna run compile --parallel",
+    "compile": "npm run clean:lib && lerna run compile",
```
Comment (sdadn) [inline package.json:84]: why is the parallel flag removed?

Comment (kenk2) [reply package.json:84]: @sdadn Parallel  is already a feature in lerna v6, so we don't really need to specify it: https://lerna.js.org/docs/lerna6-obsolete-options#--parallel

File: package.json (hunk 1 above)
Comment (kenk2) [reply package.json]: Done!

**Review by sdadn — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -4,13 +4,16 @@
   "version": "5.25.0",
   "description": "Utilities to help when developing terra modules.",
   "engines": {
-    "node": "10 || 12|| 14",
-    "npm": "6 || 9"
```
Comment (nikhitasharma) [inline package.json:8]: Are we okay going ahead and removing npm6 support based on discussion during retro?

Comment (kenk2) [reply package.json:8]: We will be, because the upgrades won't affect consumers. @nikhitasharma

**Review by adoroshk — APPROVED** [review]

**Pull request merged**